src/components/App.js

```diff
@@ -79,7 +81,12 @@ const App = () => {
       <DragDropContext onDragEnd={handleDragEnd}>
         <Droppable droppableId="app" type="list" direction="horizontal">
           {(provided) => (
-            <div className="flex px-6 pt-3 list-height scroll-x-hidden" ref={provided.innerRef} {...[provided.droppableProps]}>
+            <div
+              id="lists-wrapper"
+              className="flex px-6 pt-3 lists-wrapper overflow-x-auto"
+              ref={provided.innerRef}
+              {...[provided.droppableProps]}
+            >
```

Comment:
Whats changed here:
1. id added to identify div
2. `scroll-x-hidden` changed to `overflow-x-auto` to show horizontal scroll
3. `list-height` class renamed to `lists-wrapper`